Resources/Prototypes/Corvax/Entities/Objects/Devices/producrion.yml

```
@@ -0,0 +1,59 @@
+  description: Плата для создания техфаба синдиката.
+  components:
+    - type: Sprite
+      state: security
+    - type: MachineBoard
+      prototype: SyndicateTechFab
+
+- type: entity
+  id: GyroscopeSyndicateMachineCircuitboard
+  parent: [BaseMachineCircuitboard, BaseC3ContrabandUnredeemable]
+  name: гироскоп Синдиката (машинная плата)
+  components:   
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)